libmisc/salt.c

```diff
@@ -90,12 +90,11 @@ static void seedRNG (void)
  */
 #define MAGNUM(array,ch)	(array)[0]=(array)[2]='$',(array)[1]=(ch),(array)[3]='\0'
 #ifdef USE_BCRYPT
-/* 
- * Using the Prefix $2a$ to enable an anti-collision safety measure in musl libc.
- * Negatively affects a subset of passwords containing the '\xff' character,
- * which is not valid UTF-8 (so "unlikely to cause much annoyance").
+/*
+ * Using the Prefix $2b$, which is the preferred prefix to use for new bcrypt
+ * hahses for a long time now.
```

Comment:
Typo in "hahses". I'd just drop the comment, really.

Reply:
Fixed in rebased commit.